pkg/integration/integration_test.go

```diff
@@ -0,0 +1,15 @@
+package integration
```

Comment:
Brief package description might help with "what goes in here" in the future
```suggestion
// integration is a package for testing application routes
// it should attempt to mock as few dependencies as possible
// and simulate production application use
package integration
```